src/elements/content-explorer/utils.ts

```diff
@@ -28,21 +39,132 @@ export function useSelectedItemText(currentCollection: Collection, selectedItemI
     }, [currentCollection.items, formatMessage, selectedItemIds]);
 }
 
+// Check if the field value is empty.
+// Note: 0 doesn't represent empty here because of float type field
+export function isEmptyValue(value: ItemMetadataFieldValue) {
+    if (isNil(value)) {
+        return true;
+    }
+
+    // date, string, enum
+    if (value === '') {
+        return true;
+    }
+
+    // multiSelect
+    if (Array.isArray(value) && value.length === 0) {
+        return true;
+    }
+
+    // float
+    if (Number.isNaN(value)) {
+        return true;
+    }
+
+    return false;
+}
+
+// Check if the field values are equal based on the field types
+export function areFieldValuesEqual(
+    fieldType: MetadataFieldType,
+    value1: ItemMetadataFieldValue,
+    value2: ItemMetadataFieldValue,
+) {
+    if (isEmptyValue(value1) && isEmptyValue(value2)) return true;
```

Comment:
nit: there was a lint/prettier format to use curly brackets:
```suggestion
    if (isEmptyValue(value1) && isEmptyValue(value2)) {
        return true;
    }
```
not sure if it was removed after the eslint upgrade but we still keep it / add it back